src/freenas/usr/local/lib/middlewared_truenas/plugins/enclosure_/map.py

```diff
@@ -120,11 +140,15 @@ async def _map_enclosures(self, enclosures, slots):
             elements.append(element)
 
         info = await self.middleware.call("system.info")
+        if '-MINI-' in ["system_product"]:
```

Comment:
This is wrong and will always evaluate to `False`

Reply:
Thanks  you mean that I messed up what should of been info["system_product"] right?

Reply:
Yeah, that's what it looks like from here.